.github/workflows/release.yml

```diff
@@ -77,7 +77,7 @@ jobs:
             file_name=${chart##*/}
             chart_name=${file_name%-*}
             digest=$(awk -F "[, ]+" '/Digest/{print $NF}' < helm-push-output.log)
-            cosign sign "ghcr.io/${GITHUB_REPOSITORY}/${chart_name}@${digest}"
+            COSIGN_YES=true cosign sign "ghcr.io/${GITHUB_REPOSITORY}/${chart_name}@${digest}"
           done
         env:
           COSIGN_EXPERIMENTAL: 1
```

Comment:
maybe drop this one and replace to `COSIGN_YES=true`

Reply:
@cpanato I don't understand your comment. Can you explain what you want to replace here, I am already using COSIGN_YES ?

Reply:
can we remove the `COSIGN_EXPERIMENTAL: 1` and instead put the COSIGN_YES=true inline with the command add in the `env` section, like

```
...
            cosign sign "ghcr.io/${GITHUB_REPOSITORY}/${chart_name}@${digest}"
          done
        env:
          COSIGN_YES: "true"
```

Reply:
nvm I now understood, you wanted to replace `COSIGN_EXPERIMENTAL: 1` with `COSIGN_YES`.